documentation/src/main/asciidoc/topics/server_protocols_hotrod.adoc

```diff
@@ -45,18 +48,46 @@ clientBuilder
         .host("127.0.0.1")
         .port(11222)
     .security()
-        .ssl()
+        .authentication()
             .username("myuser")
             .password("qwer1234!");
 remoteCacheManager = new RemoteCacheManager(clientBuilder.build());
 RemoteCache<String, String> cache = remoteCacheManager.getCache("secured");
 ----
 
+==== DIGEST
+Deprecated by SCRAM and available only for backwards compatibility with
+previous versions of {brandname} server that do not support SCRAM.
+
+.DIGEST-MD5 Hot Rod client configuration
+
+[source,java]
+----
+ConfigurationBuilder clientBuilder = new ConfigurationBuilder();
+clientBuilder
+    .addServer()
+        .host("127.0.0.1")
+        .port(11222)
+    .security()
+        .authentication()
+            .saslMechanism("DIGEST-MD5")
+            .username("myuser")
+            .password("qwer1234!")
+            .realm("default");
```

Comment:
Why is `realm` spelled out in this example, even if it contains the default value, but not server name? The same happens with other examples throughout the documentation.

Reply:
I have removed all examples which use the defaults. I've also externalized all of the code examples to their own individual files.